src/encoded/static/scss/encoded/modules/_matrix.scss

```diff
@@ -15,6 +18,390 @@ $row-data-header-width: 200px;
+        height: 0;
+        border: 0;
+        overflow: hidden;
+    }
+    .collapse-body-map {
+        cursor: pointer;
+        margin: 0;
+        padding: 0;
+        width: 100%;
+        font-size: 1rem;
+        border: 0;
+    }
+    .body-list {
+        .body-list-inner {
+            text-align: center;
+        }
+    }
+    .body-list-inner {
+        li {
+            display: inline;
+            &:after {
+                content: " ";
+                word-spacing: 0em;
+                letter-spacing: 1em;
+                background:url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAUAAAAFCAYAAACNbyblAAAABGdBTUEAALGPC/xhBQAAAAlwSFlzAAAOwAAADsABataJCQAAABl0RVh0U29mdHdhcmUAcGFpbnQubmV0IDQuMC4xMkMEa+wAAAAnSURBVBhXY/Dz89MA4sNA/B9Ka4AEYQIwfBgkiCwAxjhVopnppwEApxQqhnyQ+VkAAAAASUVORK5CYII=) center center no-repeat;
+            }
+        }
+        button {
+            margin-right: 0;
+            padding: 2px 5px;
+        }
+    }
+    .body-facet {
+        .body-image-container {
+            flex: 0 1 50%;
+            @media screen and (max-width: $screen-xs-min) {
+                flex: 0 1 100%;
+            }
+        }
+        .body-list {
+            flex: 0 1 50%;
+            @media screen and (max-width: $screen-xs-min) {
+                flex: 0 1 100%;
+            }
+        }
+        .body-inset-container {
+            flex: 0 1 60%;
+            text-align: center;
+            .body-inset {
+                img {
+                    width: 55px;
+                    height: 55px;
+                }
+            }
+            .body-inset:after {
+                width: 55px;
+                height: 55px;
+            }
+        }
+        .body-list-narrow {
+            flex: 0 1 40%;
+        }
+    }
+    .body-facet :nth-child(1) {
+        order: 1;
+    }
+    .body-facet :nth-child(2) {
+        order: 2;
+    }
+    .body-facet :nth-child(3) {
+        order: 4;
+    }
+    .body-facet :nth-child(4) {
+        order: 3;
+    }
+    .clickable-diagram-container {
+        .body-list-inner {
+            .body-list-element {
+                width: auto;
+            }
+        }
+    }
+}
+
+.expand-arrows, .collapse-arrows {
+    height: 30px;
+    width: 30px;
+    position: absolute;
+    top: 43px;
+    left: 5px;
+}
+.expand-arrows {
+    fill: $Homo-sapiens;
+    @media screen and (max-width: $screen-sm-min) {
+        top: 8px;
+        fill: white;
+        height: 20px;
+        width: 20px;
+    }
+    @media screen and (max-width: $screen-xs-min) {
+        top: 3px;
+    }
+}
+.collapse-arrows {
+    fill: white;
+    top: 2px;
+    z-index: 999;
+    @media screen and (max-width: $screen-sm-min) {
+        top: 8px;
+        height: 20px;
+        width: 20px;
+    }
+    @media screen and (max-width: $screen-xs-min) {
+        top: 3px;
+    }
+}
+.body-map-expander, .body-map-collapser {
+    background: $Homo-sapiens;
+    text-align: center;
+    font-family: Mada;
```

Comment:
This is just for my personal curiosity. You and Forrest seems to have used this font. What is special about it that makes it appealing?

Reply:
I think I added the font when I was making the mouse development matrix and I'm slowly propagating it through the site on more visual types of elements. In the newspaper world, I'd say it's a "graphics" font. This probably merits more discussion with the design group, but I started to use it because I find it visually appealing and easy to read. It was also sort of exploratory, to see what users and other team members would have to say about it. Overall it's nice to have a couple fonts on a site, because they allow you to come up with more interesting designs (you can have the one text for one page element and the other for another, and that creates a distinction between the two elements without using say color or border or background or the other usual visual distinctions). Of course you don't want too many fonts either, or fonts that clash. I thought this one worked reasonably well for these purposes. Do you like the font?